packages/mobile/src/backup/BackupQuiz.tsx

```diff
@@ -27,7 +29,7 @@ const TAG = 'backup/BackupQuiz'
 const MNEMONIC_BUTTONS_TO_DISPLAY = 6
 
 // miliseconds
-const CHECKING_DURATION = 2.6 * 1000
+const CHECKING_DURATION = 1.8 * 1000
```

Comment:
Why did this const change? Also can you add a comment explaining what checking duration is

Reply:
oh i thought it felt too long. I was the one who originally set it to 2.6 but regretted it being so long.